<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="size_label">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="created_date_label">
       <property name="text">
        <string>Create Date</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="modified_date_label">
       <property name="text">
        <string>Modified Date</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="accessed_date_label">
       <property name="text">
        <string>Accessed Date</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="select_button">
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="size_edit"/>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="created_date_edit"/>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="modified_date_edit"/>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="accessed_date_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="property_label">
     <property name="text">
      <string>Property:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="dir_box">
       <property name="text">
        <string>Dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="file_box">
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="link_box">
       <property name="text">
        <string>Link</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="hidden_box">
       <property name="text">
        <string>Hidden</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="read_box">
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="write_box">
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="execute_box">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="get_fileinfo_button">
     <property name="text">
      <string>Get File Info</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
